src/app/z-index-test/page.tsx

```
@@ -0,0 +1,436 @@
+                      모달은 헤더/사이드바/모달 백드롭보다 위에 떠야 합니다.
+                    </p>
+
+                    <div className="mt-6 flex gap-2">
+                      <button className={`
+                        rounded-xl border border-border bg-card px-3 py-2
+                        text-sm
+                      `}
+                      >
+                        Cancel
+                      </button>
+                      <button className={`
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)